pkg/controllers/cstorvolumeconfig/controller.go

```diff
@@ -330,13 +331,23 @@ func (c *CVCController) createVolumeOperation(cvc *apis.CStorVolumeConfig) (*api
 	if isHAVolume(cvc) {
 		// TODO: When multiple threads or multiple CVC controllers are set then
 		// we have to revist entier PDB code path
-		var pdbObj *policy.PodDisruptionBudget
-		pdbObj, err = c.getOrCreatePodDisruptionBudget(getCSPC(cvc), poolNames)
-		if err != nil {
-			return nil, errors.Wrapf(err,
-				"failed to create PDB for volume: %s", cvc.Name)
+		if useV1beta1PodDisruptionBudget {
+			var pdbObj *policyv1beta1.PodDisruptionBudget
+			pdbObj, err = c.getOrCreateV1beta1PodDisruptionBudget(getCSPC(cvc), poolNames)
+			if err != nil {
+				return nil, errors.Wrapf(err,
+					"failed to create PDB for volume: %s", cvc.Name)
+			}
+			addV1beta1PDBLabelOnCVC(cvc, pdbObj)
+		} else {
+			var pdbObj *policyv1.PodDisruptionBudget
+			pdbObj, err = c.getOrCreateV1PodDisruptionBudget(getCSPC(cvc), poolNames)
+			if err != nil {
+				return nil, errors.Wrapf(err,
+					"failed to create PDB for volume: %s", cvc.Name)
+			}
+			addV1PDBLabelOnCVC(cvc, pdbObj)
```

Comment:
Do we need these if...else statements. Should we simply change this to v1 api and update the docs specifying the minimum K8s version required by cStor to work? This way we won't be needing to support both the versions simultaneously.

Reply:
Fair enough ✔️